scripts/create-release-objects.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# This script creates the release artifacts for the project at the specified directory
+# Usage: ./scripts/create-release-objects.sh <release-directory>
+
+# Check if release directory is specified exists
+if [ -z "$1" ]; then
+    echo "Please specify the release directory"
+    exit 1
+fi
+
+RELEASE_DIR=$1
+
+# Create YAML file which contains all policies
+RELEASE_POLICY_YAML_FILE_NAME=$RELEASE_DIR/kubescape-validating-admission-policies.yaml
+
+echo "Creating release policy YAML file $RELEASE_POLICY_YAML_FILE_NAME"
+echo "" > $RELEASE_POLICY_YAML_FILE_NAME
```

Comment:
`touch` instead?

Reply:
Fixes

Reply:
@matthyx , I reverted to `echo` because I remembered why it is better: if the file already exists, it deletes its contents.